src/private/utils.js

```diff
@@ -62,13 +62,10 @@ const getOwnKeys = getOwnPropertySymbols
 
 
 export function getOwnPropertyDescriptors(obj) {
-  const descs = {};
-
-  getOwnKeys(obj).forEach(
-    key => (descs[key] = getOwnPropertyDescriptor(obj, key))
-  );
-
-  return descs;
+  return getOwnKeys(obj).map(key => [
```

Comment:
I see the point with `Object.getOwnPropertyDescriptors`. I'll upgrade this piece of code, so implementation stays one to one with proposal interface.